### ISSUE TITLE
Response field names do not match V2 API

### ISSUE THREAD

**Issue by danjcosg** [jpalumickas/mailerlite-ruby#41]
Hi, 
First off thanks for the gem, it's better than the official one (since I can actually a make requests).

I'm wondering if there are plans to switch to the V2 API? I'm worried MailerLite will phase out V1, and the naming scheme & docs seems better for V2 anyway.

**Comment by jpalumickas** [comment]
Hey!

Good question, I just saw that they made official Ruby gem, so we need to think if it's better to support official gem or migrate this one to V2.

**Comment by danjcosg** [comment]
Yeah I've been considering which to use myself. The official gem is currently buggy to the point that it's not usable!

So for now I'll stick with this gem since I need something that works today.

Long-term though I guess it makes sense to support the official gem, depending on how open they are to feedback & collaboration. It seems a bit rushed atm
